keps/sig-node/4033-group-driver-detection-over-cri/README.md

```diff
@@ -265,231 +177,85 @@ will take precedence over cgroupDriver setting from the kubelet config (or
 `--cgroup-driver` command line flag). If the runtime does not provide
 information about the cgroup driver, then kubelet will fall back to using its
 own configuration (`cgroupDriver` from kubeletConfig or the `--cgroup-driver`
-flag). Further, the kubeletConfig field and `--cgroup-driver` flag will be
-marked as deprecated, to be dropped when support for the feature is adopted by
-CRI-O and containerd. Usage of the deprecated setting will produce a log
+flag). For beta, the kubeletConfig field and `--cgroup-driver` flag will be
```

Comment:
"will be" or "have been"?

Reply:
They were not truly deprecated in beta (except for the log message saying/lying so). Now changed the wording to

```
In beta, resorting to the fallback behavior will produce a log message like:
```